document how to perform and validate external -> embedded etcd migration (#712)
loft-sh/vcluster-docs: vcluster/configure/vcluster-yaml/control-plane/components/backing-store/etcd/deploy.mdx

There is an option to migrate from a deployed etcd to the [embedded](/vcluster/next/configure/vcluster-yaml/control-plane/components/backing-store/etcd/embedded) etcd option. Replace your current backing store with this option and upgrade your virtual cluster.

```yaml title="vcluster.yaml"
controlPlane:
  backingStore:
    etcd:
      embedded:
        enabled: true
        migrateFromDeployedEtcd: true
```

> Deploying with `migrateFromDeployedEtcd: true` keeps an external ETCD StatefulSet to perform a migration. After successful migration you should see `Successfully migrated etcd database to embedded etcd` log message in vCluster.

Reviewer: ```suggestion
Deploying with `migrateFromDeployedEtcd: true` retains the external etcd StatefulSet to perform the migration.  After a successful migration, you should see the `Successfully migrated etcd database to embedded etcd` log message in vCluster.
```

> To get rid of external ETCD after migration, remove `migrateFromDeployedEtcd: true` from `vcluster.yaml`:

Reviewer: ```suggestion
To remove the external etcd after migration, delete `migrateFromDeployedEtcd: true` from the `vcluster.yaml`:
```

> ```yaml title="vcluster.yaml after ETCD migration"

Reviewer: ```suggestion
```yaml title="vcluster.yaml after etcd migration"
```
